bedrock/mozorg/templates/mozorg/contribute/signup.html

```diff
@@ -164,7 +164,7 @@ <h2 class="section-title">{{ _('Now tell us a little about yourself:') }}</h2>
           <label class="inline" for="id_newsletter">
             {{ form.newsletter }}
             {{ form.newsletter.errors }}
-            {{ _('Sign me up for community updates') }}
+            {{ _('Sign up now for community updates') }}
```

Comment:
This doesn't affect l10n, since the new sign-up form was never used outside of en-US, see https://www.mozilla.org/en-GB/contribute/signup/ vs https://www.mozilla.org/en-US/contribute/signup/